src/ActionList/Group.tsx

```diff
@@ -58,14 +58,14 @@ export const Group: React.FC<React.PropsWithChildren<ActionListGroupProps>> = ({
       }}
       {...props}
     >
-      {title && <Header title={title} variant={variant} auxiliaryText={auxiliaryText} labelId={labelId} />}
       <GroupContext.Provider value={{selectionVariant}}>
         <Box
           as="ul"
           sx={{paddingInlineStart: 0}}
           aria-labelledby={title ? labelId : undefined}
           role={role || (listRole && 'group')}
         >
+          {title && <Header title={title} variant={variant} auxiliaryText={auxiliaryText} labelId={labelId} />}
```

Comment:
Should `auxiliaryText` be `sutitle`?

Reply:
This whole file should be gone, actually. I wonder if these got folded back during a merge.